.golangci.yml

```
@@ -18,6 +18,7 @@
 ---
 run:
   concurrency: 6
+  timeout: 2m # 1m by default
```

Our YAML linter rules requires at least 2 spaces before a comment:

```suggestion
  timeout: 2m  # 1m by default
```